products/emberjs.md

```diff
@@ -16,27 +16,44 @@ auto:
 -   npm: ember-source
 
 releases:
--   releaseCycle: "4"
+-   releaseCycle: "4.12"
+    releaseDate: 2023-04-03
     support: true
     eol: false
     latest: "4.12.0"
     latestReleaseDate: 2023-04-03
-    releaseDate: 2021-11-15
+
+-   releaseCycle: "4.8"
+    lts: true
+    releaseDate: 2022-11-28
```

Comment:
4.8.0 was released on 2022-10-17 according to the git tag date, see https://github.com/endoflife-date/release-data/blob/74489af2880e199b375cdb570e5c0663d2996421/releases/emberjs.json#LL166C4-L166C10. 2022-11-28 is corresponding to 4.9.0, so should be moved to the `lts` field.

Same remarks for 4.4 cycle.

Reply:
Fixed with 25b7ec5